.gitignore

```diff
@@ -1 +1,2 @@
+#ignore 
 node_modules/
```

Comment:
Good job including the node_modules/ in .gitignore, It helps keep your files light when you push to your remote 👍🏽

Reply:
Well done 👍